### PR TITLE
Create Project Structure

### PR DESCRIPTION
**Pull request by Maks122525**

### CODE REVIEW

File: project_depatment/project_depatment/settings.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+# Application definition
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
```
Comment (inauros) [inline project_depatment/project_depatment/settings.py:40]: your app missed

**Review by inauros — CHANGES_REQUESTED** [review]
1. in accordance with the task, modules should be replaced with packages
2. .idea folder and the whole its content is not a part of the project, please add .idea to .gitignore
3. please add all __ pycache __ directories to .gitignore as well
4. project_department/application/urls.py missed

File: project_depatment/project_depatment/settings.py (hunk 1 above)
Comment (inauros) [reply project_depatment/project_depatment/settings.py:40]: not fixed yet

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/project_depatment/.idea
+/project_depatment/project_depatment/__pycache__/
```
Comment (inauros) [inline .gitignore]: you excluded only /project_depatment/project_depatment/__pycache__/ and /project_depatment/.idea, please use wildcards to make it more general and exclude all possible pycache and .idea directories from the project.

File: project_depatment/application/static/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (inauros) [inline project_depatment/application/static/__init__.py]: static is not a package - init.py is redundant

File: project_depatment/application/templates/__init__.py [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (inauros) [inline project_depatment/application/templates/__init__.py]: templates is not a package - init.py is redundant

**Review by inauros — CHANGES_REQUESTED** [review]

File: project_depatment/application/templates/__init__.py (hunk 4 above)
Comment (Maks122525) [reply project_depatment/application/templates/__init__.py]: fixed

File: project_depatment/application/static/__init__.py (hunk 3 above)
Comment (Maks122525) [reply project_depatment/application/static/__init__.py]: fixed

File: .gitignore (hunk 2 above)
Comment (Maks122525) [reply .gitignore]: fixed

File: project_depatment/project_depatment/settings.py (hunk 1 above)
Comment (Maks122525) [reply project_depatment/project_depatment/settings.py:40]: In point 4, you need to delete the file?

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,4 @@
+**/.idea
+**/__pycache__
+
```
Comment (jimmyhollywood) [inline .gitignore:3]: Please add database files (sqlite3) to gitignore as well.

Generally, it'd be better to use the ready-to-use templates of gitignore files (examples could be found herehttps://github.com/github/gitignore)

**Review by jimmyhollywood — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 5 above)
Comment (Maks122525) [reply .gitignore:3]: fixed

**Review by jimmyhollywood — CHANGES_REQUESTED** [review]
There is not full match project structure to grow step description.  I did not see `templates` and `static` folders.

The database file is still presented in branch (db.sqlite3)

**Review by jimmyhollywood — APPROVED** [review]

**Pull request merged**